plugins/telemetry/telemetry.h

```diff
@@ -364,13 +364,19 @@ class Telemetry : public PluginBase
     /**
      * @brief Get the camera's attitude in quaternions (synchronous).
      *
+     * Note this attitude is relative to North, so in the absolute frame
```

Comment:
This is OK, but possibly more than needs to be said. I'd suggest for both changes:

> Note that the yaw component of attitude is relative to North (absolute frame).